tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/EditorMicroservice.java

```diff
@@ -1117,10 +1118,16 @@ public Response exportApps(@QueryParam("type") String exportType, @FormParam("pa
             ExportAppsRequest exportAppsRequest = new Gson().fromJson(payload, ExportAppsRequest.class);
             ExportUtils exportUtils = new ExportUtils(configProvider, exportAppsRequest, exportType);
             File zipFile = exportUtils.createZipFile();
+            String fileName = "siddhi-docker.zip";
+            if (exportType != null) {
+                if (exportType.equals(EXPORT_TYPE_KUBERNETES)) {
+                    fileName = "siddhi-kubernetes.zip";
+                }
+            }
```

Comment:
This could be improved as EXPORT_TYPE_KUBERNETES.equals(exportType)

Reply:
Fixed in https://github.com/siddhi-io/distribution/pull/794/commits/299f71bb802e9cd7c0b16f595f2247026f0b1739